### PR TITLE
Update stdlib (maybe?)

### PR DESCRIPTION
**Pull request by glennsl**
Missing modules have been fixed in bucklescript-playground (see https://github.com/BuckleScript/bucklescript-playground/issues/2)

Did the copy files and run regen script routine, but I couldn't actually get it to work locally. Am I missing a step?

### CODE REVIEW

**Comment by glennsl** [comment]
I suppose this is just the js runtime part of it, and I have to rebuild the bs.js bundle to get the external definitions perhaps?

**Comment by glennsl** [comment]
Ok, I think I got it. Just another file that had to be copied and renamed.

File: regenerateStdlibForBrowser.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ try {
   console.log(`
 ** There's an error while reading ${stdlibDir}. Is this your first time setting things up?
 You need to copy the BuckleScript standard library files from here:
-https://github.com/BuckleScript/bucklescript/tree/d21cdfbbe61afb0ae6b79735765de949409947ea/lib/js
+https://github.com/BuckleScript/bucklescript-playground/tree/master/stdlib
```
Comment (chenglou) [inline regenerateStdlibForBrowser.js]: Humm, we're using the commonjs version; this is the amd one though? How does this work?

Comment (glennsl) [reply regenerateStdlibForBrowser.js]: Oh, well it doesn't look like it actually does. I've just been testing with code that didn't need any runtime. If it does need runtime, I get "define is not defined". Hold on, I'll fix.

**Comment by glennsl** [comment]
@chenglou Ok, this seems to work better

**Comment by chenglou** [comment]
Tested and works. Thanks!

**Pull request merged**